src/main/java/com/artipie/settings/repo/CrudRepoSettings.java

```diff
@@ -41,6 +44,20 @@ public interface CrudRepoSettings {
      */
     JsonStructure value(RepositoryName name);
 
+    /**
+     * Get repository settings as yaml.
+     * @param name Repository name.
+     * @return Yaml repository settings
+     * @throws IOException On IO error
+     */
+    YamlMapping valueAsYaml(RepositoryName name) throws IOException;
+
+    /**
+     * Repositories configuration storage.
+     * @return Repo configs {@link BlockingStorage}
+     */
+    BlockingStorage repoConfigsStorage();
```

Comment:
check previous comments, these methods are not needed

Reply:
@olenagerasimova, @dgarus Please review